src/selectors.js

```diff
@@ -3,7 +3,7 @@ import { waiterModel } from './waiter-reducer'
 
 import { NAME } from './constants'
 
-export const getWaiterStore = createSelector(
+const getWaiterStore = createSelector(
```

Comment:
I didn't see this getting imported anywhere, so I removed the `export` for clarity.